bleak/__init__.py

```diff
@@ -58,6 +58,13 @@ class BleakScanner:
     """
     Interface for Bleak Bluetooth LE Scanners.
 
+    The scanner will listen for BLE advertisements, optionally filtering on advertised services or
+    other conditions, and collect a list of `BLEDevice` objects. These can subsequently be used to
```

Comment:
Needs a prefix for the link to work.

```suggestion
    other conditions, and collect a list of :class:`BLEDevice` objects. These can subsequently be used to
```